style/main.scss

```diff
@@ -76,37 +76,17 @@ $windowHeight: 400;
 
 .window {
   background: $white;
-  height: calc(100% - 40px);
 
   box-sizing: border-box;
   z-index: 1000;
+  position: absolute;
 
-  margin: 20px;
-  padding: 10px;
-  border: 6px solid $lightgray-dark;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
I remember that the reason that I started de-absoluting everything was because I was working on a `window`, but it didn't need to be full-size. I needed more flexibility.

However, that was a couple weeks ago, and I might've been mistaken. We can leave this as-is, and, worst-comes-to-worst, someone can change it _if_ they need to change it (when we add more windows)